scripts/src/scrubMessages.ts

```diff
@@ -73,7 +52,7 @@ const scrubMessages = async ({ db }: { db: Db }) => {
     },
     {
       $project: {
-        _id: "$messages.id",
+        _id: "$messages.id", // FIXME: why inconsistent usage of _id vs. id?
```

Comment:
We _could_ switch `Message` to use `_id` but it'd need some prod db migration + code cleanup. Loosely I think `_id` implies the root-level document ID but I suppose it could also make sense in an embedded document.